packages/console-types/src/types/catalog/item-manifest.ts

```diff
@@ -71,4 +71,7 @@ export const catalogItemManifestSchema = {
   type: 'object',
 } as const satisfies JSONSchema
 
-export type CatalogItemManifest = FromSchema<typeof catalogItemManifestSchema>
+export type CatalogItemManifest<
```

Comment:
isn't this a breaking change for the exported type? if so I guess this PR is semver-minor (changeset states patch)

Reply:
All the generics have the same default value they had before, so it should not be beaking